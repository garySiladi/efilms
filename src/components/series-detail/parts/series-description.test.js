// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import SeriesDescription from './series-description';

const seriesAssetMockData = {
  file: {
    url: 'u1',
  },
};
const sponsorAssetMockData = {
  file: {
    url: 'u2',
  },
};

test('SeriesDescription renders with data', () => {
  const tree : string = renderer.create(
    <SeriesDescription
      backgroundUrl="xxx"
      description="aaa"
      seriesLogoUrl={seriesAssetMockData.file.url}
      sponsorLogoUrl={sponsorAssetMockData.file.url}
      isWatchnowBtnSelected={false}
    />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
