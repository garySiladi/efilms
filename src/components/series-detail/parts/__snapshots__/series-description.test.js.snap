// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeriesDescription renders with data 1`] = `
<div
  className="series-description__parent"
>
  <div
    className="series-description"
  >
    <div
      className="series-description__series-logo series-description__flex-child"
    >
      <div
        className="image"
      >
        <div
          className="image__holder"
        >
          <img
            alt="series logo"
            className="series-description__logo-img"
            onLoad={[Function]}
            src="u1"
          />
        </div>
      </div>
    </div>
    <div
      className="series-description__description series-description__flex-child"
    >
      aaa
    </div>
    <div
      className="series-description__right-wrapper series-description__flex-child"
    >
      <div
        className="series-description__sponsor-logo"
      >
        <div
          className="image"
        >
          <div
            className="image__holder"
          >
            <img
              alt="series logo"
              className="series-description__logo-img"
              onLoad={[Function]}
              src="u2"
            />
          </div>
        </div>
      </div>
      <div
        className="series-description__watchnow-btn"
        role="button"
        to="/"
      >
        Watch now
      </div>
    </div>
  </div>
</div>
`;
